project/Easy-Travel-BD/EasyTravel.Web/wwwroot/assets/js/main.js

```
@@ -0,0 +1,218 @@
+  window.addEventListener("load", initSwiper);
+
+  /**
+   * Initiate Pure Counter
+   */
+  new PureCounter();
+
+  /**
+   * Init isotope layout and filters
+   */
+  document.querySelectorAll('.isotope-layout').forEach(function(isotopeItem) {
+    let layout = isotopeItem.getAttribute('data-layout') ?? 'masonry';
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'layout' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)